src/main/scala/com/gu/scanamo/ops/ScanamoInterpreters.scala

```diff
@@ -0,0 +1,130 @@
+package com.gu.scanamo.ops
+
+import cats._
+import cats.data.Xor
+import com.amazonaws.AmazonWebServiceRequest
+import com.amazonaws.handlers.AsyncHandler
+import com.amazonaws.services.dynamodbv2.{AmazonDynamoDB, AmazonDynamoDBAsync}
+import com.amazonaws.services.dynamodbv2.model.{UpdateItemResult, _}
+import com.gu.scanamo.request.{ScanamoDeleteRequest, ScanamoPutRequest, ScanamoUpdateRequest}
+
+import scala.concurrent.{ExecutionContext, Future, Promise}
+import scala.util.{Failure, Success}
+
+object ScanamoInterpreters {
```

Comment:
Mainly because I'm addicted to IntelliJ's 'find class', which is much less fun to use if I have to worry about the package as well as the name.
 
Am willing to be convinced otherwise, but that's my rationale for preferring the duplication/verbosity.